docs/medical-api/api-reference/document/list-documents.mdx

```diff
@@ -16,51 +16,132 @@ query, see the [Start Document Query endpoint](/medical-api/api-reference/docume
+      "meta": {
+        "versionId": "1",
+        "lastUpdated": "2023-09-01T14:31:57.025+00:00",
+        "source": "#vK8lyyAta8eKkbPO"
+      },
+      "contained": [
+        {
+          "resourceType": "Organization",
+          "id": "org522",
+          "name": "Hospital 522"
+        },
+        {
+          "resourceType": "Practitioner",
+          "id": "auth925",
+          "name": [
+            {
+              "text": "John Snow",
+              "family": "Snow",
+              "given": ["John"]
+            }
+          ]
+        }
+      ],
+      "extension": [
+        {
+          "url": "https://public.metriport.com/fhir/StructureDefinition/data-source.json",
+          "valueCoding": {
+            "system": "https://public.metriport.com/fhir/StructureDefinition/data-source.json",
+            "code": "METRIPORT"
+          }
+        }
+      ],
+      "masterIdentifier": {
+        "system": "urn:ietf:rfc:3986",
+        "value": "018a875a-3181-74e8-8aa4-40c59ed7e1b2"
+      },
+      "identifier": [
+        {
+          "use": "official",
+          "system": "urn:ietf:rfc:3986",
+          "value": "018a875a-3181-74e8-8aa4-40c59ed7e1b2"
+        }
+      ],
       "status": "current",
-      "indexed": "2019-09-07T15:50:00.000Z",
-      "mimeType": "application/xml",
-      "size": "17344007",
       "type": {
         "coding": [
           {
-            "system": "http://snomed.info/sct",
-            "code": "62479008",
-            "display": "Diagnoses"
+            "system": "http://loinc.org/",
+            "code": "34133-9",
+            "display": "Summarization of episode note"
+          }
+        ]
+      },
+      "subject": {
+        "reference": "Patient/018a5164-c0ac-79e2-99cc-9cf951c26d94",
+        "type": "Patient"
+      },
+      "date": "2023-08-15T03:05:58.090Z",
+      "author": [
+        {
+          "reference": "#org522",
+          "type": "Organization"
+        }
+      ],
+      "description": "C-CDA R2.1 Patient Record",
+      "content": [
+        {
+          "extension": [
+            {
+              "url": "https://public.metriport.com/fhir/StructureDefinition/data-source.json",
+              "valueCoding": {
+                "system": "https://public.metriport.com/fhir/StructureDefinition/data-source.json",
+                "code": "METRIPORT"
+              }
+            }
+          ],
+          "attachment": {
+            "contentType": "application/pdf",
+            "url": "http://api.metriport.com/doc-contribution/commonwell/?fileName=965f9957-34ac-442b-9a17-fa09c76c16da/018a5164-c0ac-79e2-99cc-9cf951c26d94/965f9957-34ac-442b-9a17-fa09c76c16da_018a5164-c0ac-79e2-99cc-9cf951c26d94_018a875a-3181-74e8-8aa4-40c59ed7e1b2",
+            "size": 1080416,
+            "title": "965f9957-34ac-442b-9a17-fa09c76c16da/018a5164-c0ac-79e2-99cc-9cf951c26d94/965f9957-34ac-442b-9a17-fa09c76c16da_018a5164-c0ac-79e2-99cc-9cf951c26d94_018a875a-3181-74e8-8aa4-40c59ed7e1b2",
```

Comment:
double checking these are pre-prod IDs in the examples throughout the PR?

Reply:
Yeah, those are local